src/web/screens/logout.tsx

```diff
@@ -0,0 +1,16 @@
+import { useKeycloak } from '@react-keycloak/web';
+
+import { PortalRoute } from './routeUtils';
+
+import './home.scss';
+
+function Logout() {
+  const { keycloak } = useKeycloak();
+  keycloak?.logout();
+  return <h1>Logging out...</h1>;
+}
+export const LogoutRoute: PortalRoute = {
+  path: '/Logout',
```

Comment:
nit: could we change it to lowercase? just to be consistent with other path

Reply:
changed

Reply:
Added instruction to add the route to the .env file